react/components/AutocompleteInput/hooks.ts

```diff
@@ -49,7 +49,7 @@ export const useArrowNavigation = (
           return setSelectedOptionIndex(index)
         case 'ArrowDown':
           e.preventDefault()
-          index = Math.min(selectedOptionIndex + 1, optionsLength - 1)
+          index = (selectedOptionIndex + 1) % optionsLength
```

Comment:
That's interesting but when clicking the input and pressing the arrow down right away, it gives an error :(

![image](https://user-images.githubusercontent.com/15948386/95121773-e7425300-0725-11eb-93d9-1a83b6fb9649.png)

Reply:
Thank you for bringing attention to this, I'm working on fixing it, turns out it's an issue with `index = NaN` when you press the arrow down right away.